c2cgeoportal/scaffolds/__init__.py

```diff
@@ -72,6 +73,27 @@ def pre(self, command, output_dir, vars):
 
         return ret
 
+    def _set_srid_in_vars(self, command, vars):
```

Comment:
Sorry to borrow you again but I think that this method should be after `_set_package_in_vars` who is the "main" method :-)